### PR TITLE
Space split to Comma split in combinatinalAnalysis.js

### LINKED ISSUE #291: Extra columns in Combinational Analysis due to an extra space in input
**Description of the bug**
In the Combinational Analysis, while entering the input number of columns vary because of a space at the end of the input.

**Steps to reproduce the behavior:**
1. Go to Simulator.
2. Click on Combinational Analysis and Enter "a b c "
3. The generated table has 4 columns instead of the expected 3 columns.

**Expected behavior**
4 columns

**Screenshots**

Circuitverse_Error

4th column is the error.

Yes, I am working on this issue

### PR DESCRIPTION
**Pull request by sal2701**
#### Describe the changes you have made in this pr -

I have changed the split parameter to "," from a " " so that names like "In A, In B" can be used.
In the code, a list was passed with the names as strings. This was causing a problem, so I mapped each string to an integer and sent the integer array forward and this caused no problems. i have used  random integer 7*i + 13 for each "i". The strings entered have been trimmed as well.

### Screenshots of the changes (If any) -

Comma_Input

### CODE REVIEW

**Comment by sal2701** [comment]
@satu0king please review

**Comment by jbox144** [comment]
Use four tabs to indent code :P

**Comment by sal2701** [comment]
@jbox144 what? :laughing:

File: public/js/combinationalAnalysis.js [hunk 1]
```diff
@@ -32,8 +34,11 @@ createCombinationalAnalysisPrompt=function(scope=globalScope){
 }
 function createBooleanPrompt(inputListNames,outputListNames,scope=globalScope){
 
-    inputListNames=inputListNames||(prompt("Enter inputs separated by space").split(' '));
-    outputListNames=outputListNames||(prompt("Enter outputs separated by space").split(' '));
+    inputListNames=inputListNames||(prompt("Enter inputs separated by commas").split(','));
+    outputListNames=outputListNames||(prompt("Enter outputs separated by commas").split(','));
+    outputListNamesInteger=[];
+    for (var i = 0; i < outputListNames.length; i++)
+        outputListNamesInteger[i] = 7*i + 13;//assigning an integer to the value, 7*i + 13 is random
```
Comment (satu0king) [inline public/js/combinationalAnalysis.js]: Why use something like this why can it simply be the index i itself?

Comment (sal2701) [reply public/js/combinationalAnalysis.js]: Idk I just felt that the index i might just repeat somewhere. No good reason. Just a random feeling, so I put in this random expression

Comment (satu0king) [reply public/js/combinationalAnalysis.js]: Okay

**Pull request merged**